dracclient/client.py

```diff
@@ -753,6 +753,55 @@ def list_raid_controllers(self):
         """
         return self._raid_mgmt.list_raid_controllers()
 
+    def list_raid_cntrl_settings(self, by_name=False):
+        """List the RAID configuration settings
+
+        :param by_name: Controls whether returned dictionary uses RAID
+                        attribute name or instance_id as key.
+        :returns: a dictionary with the RAID settings using InstanceID as the
+                  key. The attributes are either RAIDEnumerableAttribute,
+                  RAIDStringAttribute objects.
+        :raises: WSManRequestFailure on request failures
+        :raises: WSManInvalidResponse when receiving invalid response
+        :raises: DRACOperationFailed on error reported back by the DRAC
+                 interface
+        """
+        return self._raid_mgmt.list_raid_cntrl_settings(by_name)
+
+    def set_raid_cntrl_settings(self, settings, raid_fqdd):
```

Comment:
Would you mind changing the above method name to:
set_raid_controller_settings()?

Reply:
sure